platform/andes/ae350/objects.mk

```diff
@@ -0,0 +1,14 @@
+#
+# SPDX-License-Identifier: BSD-2-Clause
+#
+# Copyright (c) 2019 Western Digital Corporation or its affiliates.
+#
+# Authors:
+#   Zong Li <zong@andestech.com>
+#
+
+platform-objs-y += trigger.o plicsw.o plmt.o platform.o
+
+ifdef AE350_ENABLED_HART_MASK
+platform-genflags-y += -DAE350_ENABLED_HART_MASK=$(AE350_ENABLED_HART_MASK)
+endif
```

Comment:
Do you really need AE350_ENABLED_HART_MASK ? If not required then please remove it.

We had added FU540_ENABLED_HART_MASK compile-time option for SiFive FU540 because at that time U-Boot was not SMP capable. After U-Boot SMP support was merged, we have stopped using FU540_ENABLED_HART_MASK on SiFive FU540 as well.

If you have M-mode only HARTs (i.e. HARTs without MMU) which cannot run Linux then you can force disable such HARTs directly in disabled_hart_mask without any compile-time AE350_ENABLED_HART_MASK option.

Reply:
We possibly not support u-boot, so we tend to keep.

Reply:
Okay, no problem.

Reply:
Sorry, we change our mind.

We fellow your advice.